src/components/colors/color.css

```diff
@@ -0,0 +1,67 @@
+.text-primary{
+    color: var(--primary-color);
+}
+.text-secondary{
+    color: var(--secondary-color);
+}
+.text-tertiary{
+    color: var(--tertiary-color);
+}
+.text-accent{
+    color: var(--accent-color);
+}
+.text-danger{
+    color: var(--danger-text-color);
+}
+.text-success{
+    color: var(--success-text-color);
+}
+.text-info{
+    color: var(--info-text-color);
+}
+.text-warning{
+    color: var(--warning-text-color);
+}
+.text-light{
+    color: var(--light-color);
+}
+.text-dark{
+    color:var(--dark-color)
+}
```

Comment:
A suggestion is to add gray text as a class here ( .text-gray).
This was naming convention also stays similar